ams_flutter/lib/features/onboarding/presentation/widgets/button.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:ams_flutter/core/constants/app_colors.dart';
+import 'package:ams_flutter/core/constants/app_text_styles.dart';
+import 'package:flutter/material.dart';
+
+class button extends StatelessWidget {
```

Comment:
The name of the widget should follow Camelcase convention. When declaring classes, the name should start with a capital letter. Moreover, the Button name is too vague, rename it to something more specific